pyanaconda/modules/payloads/source/rpm_ostree/rpm_ostree.py

```diff
@@ -76,12 +77,10 @@ def network_required(self):
 
         :return: True or False
         """
-        # TODO: Implement this method
-        return False
+        return not (self.configuration.url and self.configuration.url.startswith("file://"))
```

Comment:
Why not extend `has_network_protocol` with an option to include the ostree url styles?

(Alternatively, the import is unnecessary now and will fail the tests, I think.)